### PR TITLE
EFRS-943 - Updated local start and readme

### PR DESCRIPTION
**Pull request by azanorth**
**Task** - [Update local start and readme](https://jira.exadel.com/browse/EFRS-943)

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# CompreFace
+
+## Local start all in prod mode
+
+Run script to start docker container:
+
+- `./scripts/start.sh` for unix OS
```
Comment (pospielov) [inline dev/README.md:7]: As I know this scripts are needed only for UI developers.
Backend team just start it with docker-compose up --build - see https://github.com/exadel-inc/CompreFace/tree/develop#getting-started-for-contributors - probably we just need to move it here for backend develoeprs

File: dev/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+(only for first time run)
+
+- Run `cd ui`
+- Run `npm install`
+
+- Run `cd ../dev`
+- Run `start.sh` to start docker container with backend and UI application in live reload mode
+
+Go to http://localhost:4200/ and you can sign up with any credentials
```
Comment (pospielov) [inline dev/README.md:22]: As I remember we also wanted to move here "Tips for Windows" and rename it that this is not just for windows

File: dev/README.md (hunk 1 above)
Comment (azanorth) [reply dev/README.md:7]: Could you please clarify what are you mean by talking about `need to move it here `?

File: dev/README.md (hunk 2 above)
Comment (azanorth) [reply dev/README.md:22]: This is another .readme file is just to set up for the backend side, and also nothing note about this in the description of this task

Comment (isobolev) [reply dev/README.md:22]: steps in "Tips for Windows" will no longer be relevant after removing start-dev.sh. Please remove that info in general readme, and list in dev readme all relevant info for local start

File: dev/start.sh [hunk 3]
```diff
@@ -1,4 +1,3 @@
-git checkout develop
-git pull
-#sed -i "s|%COMMIT_HASH%|$(git rev-parse --short HEAD)|g" /ui/src/app/features/footer/footer.component.html
-docker-compose -f docker-compose.yml up --build --scale compreface-postgres-db=0
+#sed -i "s|%COMMIT_HASH%|$(git rev-parse --short HEAD)|g" ui/src/app/features/footer/footer.component.html
+docker-compose -f docker-compose.yml -f docker-compose.dev.ui.yml up --build &
```
Comment (isobolev) [inline dev/start.sh:2]: just to check this config is working, did you try to start with this config on a fresh docker? (after docker system prune)

Comment (azanorth) [reply dev/start.sh:2]: Yes, this config is working

File: dev/README.md (hunk 2 above)
Comment (azanorth) [reply dev/README.md:25]: Fixed

File: dev/start.sh (hunk 3 above)
Comment (isobolev) [reply dev/start.sh:2]: once we check that steps in dev readme also work on windows, start.sh can also be removed

**Review by isobolev — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -90,45 +90,9 @@ CompreFace:
  - Includes a UI panel with roles for access control
  - Starts quickly with one docker command
 
+## To get started
 
-## Getting started
```
Comment (pospielov) [inline README.md:94]: We shouldn't move instructions for non developers from here...

File: README.md [hunk 5]
```diff
@@ -90,45 +90,9 @@ CompreFace:
...
-2. Make sure all your containers are down: _$ docker ps_
-3. In case some containers are working, they should be stopped: _$ docker-compose down_
-4. Clean all local datebases and images: _docker system prune --volumes_
-5. Last line in /dev/start.sh file change to _docker-compose -f docker-compose.yml up --remove-orphans --build_
-6. Go to Dev folder cd dev
-7. Run _sh start.sh_ and make sure http://localhost:8000/ starts
-8. Stopped all containers: $ docker-compose down
-9. Run _sh start--dev.sh_ and make sure http://localhost:4200/ starts
-
+[Contribution guidelines to build and start the project](dev/README.md)
```
Comment (azanorth) [inline README.md:95]: > We shouldn't move instructions for non developers from here...
This instruction was moved to the dev folder

File: README.md [hunk 6]
```diff
@@ -90,45 +90,33 @@ CompreFace:
...
+#### To get started:
+
 1. Install Docker and Docker-Compose
 2. Download the archive from our latest release: https://github.com/exadel-inc/CompreFace/releases
 3. Unzip the archive
 4. Run command: _docker-compose up -d_
-5. Open in your browser: http://localhost:8000/login
 
-#### Getting started for Contributors:
+### To get started (Windows):
```
Comment (pospielov) [inline README.md:102]: Please add one # here

Comment (azanorth) [reply README.md]: Fixed

**Review by pospielov — APPROVED** [review]

**Pull request merged**